components/hypercert/hypercert-window.tsx

```diff
@@ -59,8 +59,8 @@ const HypercertWindow = ({
 
   return (
     <Link href={`/hypercerts/${hypercertId}`}>
-      <article className="transition-transform duration-300 hover:-translate-y-2 relative group bg-black/10 rounded-lg overflow-hidden">
-        <div className="h-[320px] w-full relative p-1">
+      <article className="transition-transform duration-300 hover:-translate-y-2 relative group bg-accent rounded-lg overflow-hidden pt-3">
+        <div className="h-[260px] w-full relative p-1">
```

Comment:
Should we be using rem?
```suggestion
        <div className="h-[16.25rem] w-full relative p-1">
```

Reply:
I'm for use of rems, however afaik rems are useful when we want things to scale based on some root size, like font-size. My thinking was the cards should be 260px tall no matter what... but I could go either way - just more work to calculate.